### PR TITLE
Update to 1.0.0 add Home folder plug and Desktop icon

### PR DESCRIPTION
**Pull request by geekgonecrazy**
Found your snap when looking to get robomongo snapped.  So figured i'd contribute.  

* I added home folder support since I use .ssh keys a lot.  
* Updated to 1.0.0 since it was released
* Added Desktop icon

### CODE REVIEW

File: snap/gui/robomongo.desktop [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Robomongo
+Comment=Rocket.Chat Native Cross-Platform Desktop Application via Electron.
```
Comment (frankban) [inline snap/gui/robomongo.desktop]: Leftover from a copy/paste?

File: snapcraft.yaml [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
+summary: Robomongo
 description: Native and cross-platform MongoDB manager
 confinement: strict
 
 apps:
     robomongo:
-        command: bin/robomongo.sh
-        plugs: [network, unity7]
+        command: env LC_ALL=C QT_XKB_CONFIG_ROOT="$SNAP/usr/share/X11/xkb" desktop-launch robomongo
+        plugs: [network, unity7, home]
```
Comment (frankban) [inline snapcraft.yaml]: Thanks for adding home! Please let's keep these in alphabetical order.

**Review by frankban — APPROVED** [review]
This is great Aaron, thanks for this! 
It looks very good with a couple of minor changes.

File: snap/gui/robomongo.desktop (hunk 1 above)
Comment (geekgonecrazy) [reply snap/gui/robomongo.desktop]: Yes :joy: sorry! Fixed

File: snapcraft.yaml (hunk 2 above)
Comment (geekgonecrazy) [reply snapcraft.yaml]: Reordered

**Comment by frankban** [comment]
Thanks for the quick changes!

**Pull request merged**